### PR TITLE
Text Embeddings Template

### PR DESCRIPTION
**Pull request by scottjlee**
Adds README and compute config for a new workspace template for computing text embeddings.

With https://github.com/anyscale/product/pull/26521

### CODE REVIEW

File: templates/text-embeddings/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+![Embeddings Computation Overview](diagram.jpg "Embeddings Computation Overview")
+
+## Running the code
+The provided script will work as-is; no need to modify anything the underlying script, unless you want to customize the logic to meet your specific requirements.
+
+Sample usage:
+```
+# Generate emeddings for a sample single-file parquet file containing 10 rows from the `falcon-refinedweb` HuggingFace dataset, using 1 GPU.
+python ray_embeddings.py \
```
Comment (gvspraveen) [inline templates/text-embeddings/README.md:18]: where is the python file?

Comment (scottjlee) [reply templates/text-embeddings/README.md:18]: It's in the docker image used for the cluster environment. We want to keep this code private, so it is not included in this public repo.

Comment (gvspraveen) [reply templates/text-embeddings/README.md:18]: can you create a wrapper python file here which takes params and invokes proprietary script in docker image. Seems odd to ask users to directly invoke our python file in docker image

Comment (scottjlee) [reply templates/text-embeddings/README.md:18]: yes, the relevant files (python file, ipynb, sample files, etc) are included in the `default` directory.

**Review by gvspraveen — APPROVED** [review]

File: templates/text-embeddings/README.md (hunk 1 above)
Comment (gvspraveen) [reply templates/text-embeddings/README.md:18]: Awesome thanks for confirming

**Pull request merged**